context.go

```diff
@@ -51,8 +55,18 @@ func WithTraceparent(ctx context.Context, tp *tracing.Traceparent) context.Conte
 	return context.WithValue(ctx, traceheaderKey, tp)
 }
 
-// GetTracing returns the tracing id from the context if it exists.
+// GetTraceparent returns the tracing id from the context if it exists.
```

Comment:
Maybe `GetTraceParent`?

Reply:
Traceparent is always considered as a single word in https://www.w3.org/TR/trace-context/#traceparent-header